ocaml/idl/datamodel_cluster_host.ml

```diff
@@ -31,7 +31,10 @@ let create =
     ()
 
 let destroy =
-  call ~name:"destroy" ~doc:"Remove a host from an existing cluster."
+  call ~name:"destroy"
+    ~doc:
+      "Remove a host from an existing cluster. This operation is allowed even \
+       if a cluster host is not enabled."
```

Comment:
Is it "a cluster host" or "the cluster host" (subject of this call)?

Reply:
It will be the cluster host whose uuid match the parameter of this call. Might be a good idea to use "the" here I guess?